source/_docs/relaunch.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Relaunch Existing Pantheon Site
+description: Take a new site live by moving custom domains from one Site Dashboard to another, with minimal HTTPS interruptions.
+tags: [dns]
+categories: []
+---
+Sites are considered launched on Pantheon once traffic is routed through custom domain(s). Relaunching a previously launched site is done by rerouting traffic from the existing Site Dashboard to an entirely new Site Dashboard.
+
+<div class="alert alert-info">
+<h4 class="info">Note</h4>
+<p markdown="1">The relaunch process applies exclusively to live sites already hosted on Pantheon. Otherwise, refer to [Launch Essentials](/docs/guides/launch/).</p></div>
+
+## Before You Begin
+- Log in to the new Site Dashboard on Pantheon
+- Open a second tab for the old Site Dashboard on Pantheon
+- In a third tab, log in to the domain's DNS service provider (e.g., Cloudflare, Amazon Route 53, etc.)
+- Examine existing records pointing to Pantheon
+  {% include("content/standard-dns-config.html") %}
+- Lower the TTL of existing DNS records to minimize impact of upcoming DNS changes
+  {% include("ttl.twig") %}
+
+## Roles & Permissions
+The permission to manage billing and plans is granted only to the role of **Site Owner** / **Organization Administrators**. Other roles do not have access as described on this page.
+
+<div class="alert alert-info">
+<h4 class="info">Note</h4>
+<p markdown="1">If you need to assume site and billing ownership, the current Site Owner must [transfer it to you directly](/docs/site-billing#transfer-ownership-and-billing-for-this-site).</p></div>
+
+## Relaunch Procedure
+1. In the new Site Dashboard, [upgrade the site from free to a paid plan](/docs/site-plan/#change-site-plan).
+2. In the old Site Dashboard, delete all domains affected by the relaunch, such as `example.com` and `www.example.com`:
+
+  **<span class="glyphicons glyphicons-cardio"></span> Live** > **<span class="glyphicons glyphicons-global"></span> Domains / HTTPS** > **Details** > **Remove Domain**
+
+
+3. In the new Site Dashboard, connect custom domains affected by the relaunch, such as `example.com` and `www.example.com`:
```

Comment:
We should explicitly tell them not to update DNS yet, because when connecting a new domain they will be prompted to do so. (might even be appropriate to use the red warning note?)